### ISSUE TITLE
Not picking up special characters in output for numbers, or accented characters.

### ISSUE THREAD

**Issue by richard-scryber** [richard-scryber/scryber.core#124]
Based on your second sample, I was able to upgrade my code to your latest version. I didn't specify explicit with or height on img element and... **it works as expected !** Thanks. Just note that I already test this scenaro with the version 5.0.7 of the lib and the image was not well resized (but no crash).

For use the image using CSS background properties, I had tried but stop me if I'm wrong, but you can't bind the data model on the css (containing the image data).

Since the updgrade to the latest version (which is a beta, do you know when you will release a stable version ?), the binding text doesn't display special characters. Could you just indicates to me how to fix it in order to conclude fully my problem ?
First template lines: 
```
<?xml version='1.0' encoding='utf-8' ?>
<html lang='fr' xmlns='http://www.w3.org/1999/xhtml'>
```

Exemple of display I get 
![image](https://github.com/richard-scryber/scryber.core/assets/2728729/f2637824-a1f5-47f1-96db-6c73456731ab)

I expect the € symbol. I have the same behaviour for é è ê, etc. I didn't have this behaviour with the 5.0.7 version.

_Originally posted by @zives in https://github.com/richard-scryber/scryber.core/issues/123#issuecomment-1888675528_

**Comment by richard-scryber** [comment]
Hi @zives.

I ran a test with a template `Scryber.Core.UnitTests.Binding.HtmlBinding_Test.BindXHtmlContent_CulturalValue` on version 6.0.3.2 and the string values _were working for me_ within the template and also bound values.

However, I have updated the expressive string function so it now accepts a culture string as the last argument and released as 6.0.4-beta. So you should be able to bind numeric values directly

```html
<span>{{string(Model.PrixUnitaire, 'C', 'fr-FR')}}</span>
```

Hopefully this will solve your issue, but let me know. Only other thing I can think of is the 'euro' symbol is not 

Richard

[HtmlBoundContent_Cultural.pdf](https://github.com/richard-scryber/scryber.core/files/13920055/HtmlBoundContent_Cultural.pdf)

**Comment by richard-scryber** [comment]
Just seen your update - glad it is working for you, and I will close both.
